content/applications/sales/crm/acquire_leads/convert.rst

```diff
@@ -2,66 +2,121 @@
 Convert leads into opportunities
 ================================
 
-The system can generate leads instead of opportunities, in order to add
-a qualification step before converting a *Lead* into an
-*Opportunity* and assigning to the right sales people.
-
-You can activate this mode from the CRM Settings. It applies to all your sales
-channels by default. But you can make it specific for specific channels
-from their configuration form.
+*Leads* act as qualifying steps before an opportunity is created. This provides additional time
+before a potential opportunity is assigned to a sales person.
 
 Configuration
 =============
 
-For this feature to work, go to :menuselection:`CRM --> Configuration --> Settings`
-and activate the *Leads* feature.
+To activate the *leads* setting, navigate to :menuselection:`CRM app --> Configuration -->
+Settings`, and check the box labeled, :guilabel:`Leads`. Then, click :guilabel:`Save`.
 
-.. image:: convert/convert01.png
+.. image:: convert/convert-leads-leads-setting.png
     :align: center
+    :alt: Leads setting on CRM configuration page.
 
-You will now have a new submenu *Leads* under *Leads* where they
-will aggregate.
+Activating this feature adds a new menu, :guilabel:`Leads`, to the header menu bar at the top of the
+screen.
 
-.. image:: convert/convert02.png
+.. image:: convert/convert-leads-leads-menu.png
     :align: center
+    :alt: Leads menu on CRM application.
+
+Once the *leads* setting has been activated, it applies to all sales teams by default. To turn off
```

Comment:
I would capitalize Leads here since it's the name of the setting. Up to you though.
```suggestion
Once the *Leads* setting has been activated, it applies to all sales teams by default. To turn off
```